scripts/send-parties-transaction.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-# Argument 1: is the query selector for the parties
-# Argument 2: is the parties json
+# Argument 1: is the query selector for the zkapp_command
+# Argument 2: is the zkapp_command json
 # Argument 3: is the GraphQL URI
 
 set -x
 
 curl \
   -X POST \
   -H "Content-Type: application/json" \
-  -d "$(printf '{ "query": "mutation($input: SendZkappInput!) { sendZkapp(input: $input) %s }" , "variables": { "input": { "parties": %s } } }' "$1" "$2")" $3
+  -d "$(printf '{ "query": "mutation($input: SendZkappInput!) { sendZkapp(input: $input) %s }" , "variables": { "input": { "zkapp_command": %s } } }' "$1" "$2")" $3
```

Comment:
don't forget to fix the casing here too!

Reply:
Done, and changed file name to `send-zkapp-transaction.sh`.